packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -55,19 +47,20 @@ const isInvalidFileName = (fileName: string): boolean => {
  * Creates a new Fluid file.
  * Returns resolved url
  */
-export async function createNewFluidFile(
-    getStorageToken: InstrumentedStorageTokenFetcher,
-    newFileInfo: INewFileInfo,
-    logger: ITelemetryLogger,
-    createNewSummary: ISummaryTree | undefined,
-    epochTracker: EpochTracker,
-    fileEntry: IFileEntry,
-    createNewCaching: boolean,
-    forceAccessTokenViaAuthorizationHeader: boolean,
-    isClpCompliantApp?: boolean,
-    enableSingleRequestForShareLinkWithCreate?: boolean,
-    enableShareLinkWithCreate?: boolean,
-): Promise<IOdspResolvedUrl> {
+export async function createNewFluidFile(...args: CreateNewFileArgs): Promise<IOdspResolvedUrl> {
+    const [
```

Comment:
I think it is unnecessary to create interfaces for function arguments. We would end up with a lot of interfaces which could be avoided? We can just have normal arguments and maybe add a line explaining them in the comment?

Reply:
Thanks, I removed the types for arguments.